templates/aws/support.html

```diff
@@ -1,144 +1,158 @@
+        </p>
+        <p>
+          24x7 Support is available for the Ubuntu operating system (infrastructure) and can be extended to include application support for <a href="/support#apps-support">key open source applications</a>.
+        </p>
+        <p>
+          See more details at <a href="/support">https://ubuntu.com/support</a>.
+        </p>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <h2 class="u-sv3">Features</h2>
-    <table class="p-table">
-      <thead>
-        <tr>
-          <th style="width: 50%;">Feature</th>
-          <th>Ubuntu</th>
-          <th>Ubuntu Pro</th>
-          <th>Ubuntu Pro with Support</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <th>Security patches for 2,300+ Ubuntu infrastructure packages</th>
-          <td>Yes</td>
-          <td>Yes</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>Security patches for 28,000+ open source packages, including Apache Kafka, MongoDB, RabbitMQ, Redis and NodeJS and key language ecosystems including Java, Python, Ruby and Javascript/Node.js</th>
-          <td>No</td>
-          <td>Yes</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>Livepatch for instant kernel security and higher uptimes</th>
-          <td>No</td>
-          <td>Yes</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>FIPS 140-2 and Common Criteria EAL2 certified components</th>
-          <td>No</td>
-          <td>Yes</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>Hardening profiles, including CIS and DISA STIG, with CIS and other Shell security recommendations for production environments built into hardened images in the gallery</th>
-          <td>No</td>
-          <td>Yes</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>Landscape fleet management</th>
-          <td>No</td>
-          <td>On request</td>
-          <td>On request</td>
-        </tr>
-        <tr>
-          <th>Maintenance period</th>
-          <td>5 years</td>
-          <td>10 years</td>
-          <td>10 years</td>
-        </tr>
-        <tr>
-          <th>Knowledge Base</th>
-          <td>No</td>
-          <td>No</td>
-          <td>Yes</td>
-        </tr>
-        <tr>
-          <th>Canonical-backed 24x7 Support with enterprise grade SLA, with possibilities to integrate into existing Azure support workflows</th>
-          <td>No</td>
-          <td>No</td>
-          <td>Yes</td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-</section>
+  <section class="p-strip is-deep u-no-padding--top">
+    <div class="row">
+      <hr />
+      <div class="u-fixed-width">
+        <h2 class="u-sv3">Features</h2>
```

Comment:
We are moving away from using `u-sv3`. Could you use `p-section` or one of its variants instead